api/src/main/kotlin/team/backend/common/entity/BaseEntity.kt

```diff
@@ -1,4 +1,4 @@
-package team.backend.infrastructure
+package team.backend.common.entity
```

Comment:
I think it would be better to modify the location.

__AS-IS__

- common > entity

__TO-BE__

- e.g domain > entity > Job.kt, Subejct.kt `or` domain > entity > common > xxx.kt
- e.g domain > entity > base > BaseEntity.kt `or` domain > entity > BaseEntity.kt

Reply:
@mangchhe Also I think TO-BE would be better.

__AS-IS__

![1](https://github.com/withDogecoin/chaterview/assets/47518272/96404368-2d2c-45d0-8152-945e709cc2eb)

__TO-BE__

- domain
  - member
     - query
     - command
     - entity
     - ...
- infrastructure
  - same above

Have to go too deep to know which domains you have.

Reply:
I applied.